Airline_Web_API/Airline_Web_API/Controllers/AirportsController.cs

```diff
@@ -32,9 +34,18 @@ public async Task<ActionResult<IEnumerable<Airport>>> GetAirports([FromQuery] st
                 var airportsByCity = airports.Where(airport => airport.City.Contains(value));
                 airports = airportsByName.Union(airportsByCity);
             }
-            return await airports
-                .OrderBy(airport => airport.City)
-                .ToListAsync();
+
+            var airportsSearchResults = await airports.OrderBy(airport => airport.Id).ToListAsync();
+
+            var config = new MapperConfiguration(config =>
+            {
+                config.CreateMap<Airport, AirportViewModel>();
+            });
+
+            var mapper = new Mapper(config);
```

Comment:
As this does not change between requests you can move it to static constructor and use static mapper in code.